docs-src/content/getting-started/_index.md

```diff
@@ -39,10 +39,12 @@ The SDK communicates with a Cubic Server instance using [gRPC](https://grpc.io).
 4. Start the cubic service
 
     ```bash
-    docker run -p 2727:2727 -p 8080:8080 --name cobalt cubicsvr-demo-en_us-16
+    docker run -p 2727:2727 -p 8080:8080 --name cobalt -v usage:/usage cubicsvr-demo-en_us-16
     ```
 
-    That will start listening for grpc commands on port 2727 and http requests on 8080, and will stream the debug log to stdout.  (You can replace `--name cobalt` with whatever name you want.  That just provides a way to refer back to the currently running container.)
+    That will start listening for grpc commands on port 2727 and http requests on 8080, and will stream the debug log to stdout.  (You can replace `--name cobalt` with whatever name you want.  That just provides a way to refer back to the currently running container.) 
+
+    The command also names the volume to which usage data is being written so you can track the amount of audio Cubic Server has processed, and submit the usage log to Cobalt if you have a volume-based license. That volume will persist even when the container is restarted; run `docker volume inspect usage` to see its location on the host machine.
```

Comment:
Are we using docker volumes or are we using bind mounts?

Including links to docker documentation for [bind mounts](https://docs.docker.com/storage/bind-mounts/) or [volumes](https://docs.docker.com/storage/volumes/) might be nice.